src/misc.c

```diff
@@ -1256,6 +1256,46 @@ cr_write_to_file(GError **err, gchar *filename, const char *format, ...)
     return ret;
 }
 
+gchar *cr_read_from_file(GError **err, gchar *filename, size_t *file_size) {
```

Comment:
How is this different from `g_file_get_contents()`?

Reply:
Wow, I feel stupid.  It's exactly the same.  I've just pushed a commit that remove cr_read_from_file and uses g_file_get_contents instead.